### PR TITLE
add tags to organisation object on getDetails

### PR DESCRIPTION
**Pull request by jay-meister**
Improve query to include orgs when no primary user or challenges or tags using OUTER JOINS

Adds orgs_id to the getBy function

Updates package number

### CODE REVIEW

**Review by mk4111 — APPROVED** [review]
LGTM!

**Comment by jay-meister** [comment]
hmm seems like it still needs some work: https://github.com/emfoundation/ce100-app/issues/395#issuecomment-264144681

Will reassign when ready.

**Comment by jay-meister** [comment]
Have added ordering by:
- first challenge date created 
- then by tag name (alphabetically)

File: lib/queries/orgs/getDetails.js [hunk 1]
```diff
@@ -6,23 +6,24 @@ module.exports = function (orgId) {
...
-      'JOIN tags_challenges',
-      'ON challenges.id = tags_challenges.challenges_id',
-      'JOIN tags ON tags_challenges.tags_id = tags.id',
+      'SELECT challenges.title AS chal_title, challenges.id AS chal_id, challenges.description AS chal_description, tags.name AS tag_name, tags.id AS tag_id, tags.active AS tag_active, challenges.active AS chal_active',
+      'FROM CHALLENGES',
+      'LEFT OUTER JOIN tags_challenges ON challenges.id = tags_challenges.challenges_id',
+      'LEFT OUTER JOIN tags ON tags_challenges.tags_id = tags.id',
       'WHERE challenges.org_id = ' + orgId,
-      'AND tags.active = true AND challenges.active = true',
+      'AND (tags.active IS NOT FALSE)',
```
Comment (mk4111) [inline lib/queries/orgs/getDetails.js:24]: this is cool!! What's the difference between this and `tags.active = true`?

**Review by mk4111 — APPROVED** [review]

File: lib/queries/orgs/getDetails.js (hunk 1 above)
Comment (jay-meister) [reply lib/queries/orgs/getDetails.js:24]: Yeah sorry I thought I had left a comment at the top. Maybe it got lost..

If a challenge didn't have any tags attached to it, the tags.active column was set to `NULL`, so filtering by `tags.active = true` completely removed those challenges from the results.

**Review by SimonLab — APPROVED** [review]
Nice query :wink:,  Look good :+1:

**Pull request merged**